packages/saber/lib/utils/validateConfig.js

```diff
@@ -103,8 +103,10 @@ module.exports = (config, { dev }) => {
     // Always use  / in dev mode
     result.build.publicUrl = '/'
   } else {
-    // Ensure that build.publicUrl ends with slash
-    result.build.publicUrl = result.build.publicUrl.replace(/\/?$/, '/')
+    // Ensure that build.publicUrl ends and starts with slash
+    result.build.publicUrl = result.build.publicUrl
+      .replace(/^\/?/, '/') // starting slash
```

Comment:
publicUrl can be full url, i.e. starting with `http`

Reply:
Hmm, we need to avoid values like `mypath/` though, as those cause assets not to load properly. Maybe check if it starts with `https?://` and then decide whether or not to prepend a `/`?

Reply:
a statement should be fine.